LEAF_Request_Portal/sources/FormEditor.php

```diff
@@ -591,25 +591,68 @@ public function setFormType($categoryID, $input){
         return $result;
     }
 
-    public function setFormDestructionAge(string $categoryID, int $input): int|null {
-        if ($input === 0) {
-            $input = null;
+    /**
+     * Create age (days) for destruction records
+     *
+     * @param string $categoryID - category having its destructionAge set
+     * @param int|null $input - number of days to mark a record for destruction
+     *
+     * @return int|null
+     */
+    public function setFormDestructionAge(string $categoryID, int $input = null): int|null {

```

Comment:
Need the return value to be an array/json  Ask Jamie for the setup.

Reply:
3822 was merged into this since I needed the back-end changes, but we will address these.